content/blog/2022/job-product-community-lead/index.md

```diff
@@ -0,0 +1,50 @@
+
+## Why a new role?
+
+When 2i2c began a year ago, we hired a team of engineers that had experience in cloud infrastructure, Jupyter, and open source ecosystems. In that time, we've built out the foundation of an infrastructure platform to offer community infrastructure for interactive computing that respects the [Right to Replicate](/content/right-to-replicate/).
+
+However, solely providing infrastructure is not enough. Working with the modern open source stack, using the cloud to its advantage, and bringing these tools into specific domains requires a lot of extra experience and expertise. We believe that 2i2c is in a good position to **provide guidance and support to leaders in the communities that we serve**, allowing them to get up-to-speed with the infrastructure more effectively so that they can have an impact.
+
+Moreover, we've also learned that it's crucial to **develop infrastructure in collaboration with the communities that you're serving**. A team of engineers tends to focus on code and infrastructure, and having a role that focuses their efforts on connecting with communities will give them an excellent perspective on what those communities need.
+
+We hope that this role will combine these two aspects to create a culture of learning, sharing, and guiding between 2i2c's team and the communities that we work with.
+
+## What are similar roles?
+
+We tried to find similar roles in the private and non-profit sector, but couldn't find anything that was a perfect fit. However, there were a few close matches.
+
+This role is kind-of like a **product manager** - their job is to understand the communities that we work with, and to use this to help us make decisions about what to build, how to present it, and how to engage with stakeholders.
+
+This role is also kind-of like a **customer success manager** or a **developer advocate** - they will guide and teach those in the communities we serve - particularly community leaders that go on to teach others - how to use our infrastructure most-effectively.
+
+In fact, the closest role we could find is a relatively new one: the [**Research Applications Manager**](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles/ram.html). This is a role that has been pioneered by the Turing Institute, and is similarly described as a connector that brings together many perspectives and encourages a participatory, team-based approach to research.
+
+However, the reality is that this role will ultimately get to shape the nature of this work within 2i2c themselves. As a young organization, there is a lot of flexibility for creativity and experimentation in bringing new skillsets into our organization. We hope that the person who fills this role will be excited about growing a culture of team-based approaches to our engineering and collaborations, and to share this culture with the communities that we serve.
```

Comment:
"However, the reality is that the person in this role will ultimately get to shape the nature of their work within 2i2c.